### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by saschagrunert**
- github.com/containerd/go-runc:
  v0.0.0-20190226155025-7d11b49dc076 → v0.0.0-20190603165425-9007c2405372

- github.com/containerd/project
  v0.0.0-20190306185219-831961d1e0c8 → v0.0.0-20190513184420-7fb81da5e663

- github.com/containerd/ttrpc
  v0.0.0-20180920185216-2a805f718635 → v0.0.0-20190613183316-1fb3814edf44
  Required code changes, since `onclose` is now an option:
  https://github.com/containerd/ttrpc/commit/ba15956d22fc5bfe9dd6caaefcc013cd29da38b7

- github.com/godbus/dbus:
  v0.0.0-20181101234600-2ff6f7ffd60f → v0.0.0-20190623212516-8a1682060722

- github.com/kr/pty:
  v1.1.5 → v1.1.8

- google.golang.org/grpc:
  v1.21.0 → v1.22.0

- golang.org/x/sys:
  v0.0.0-20190624142023-c5567b49c5d0 → v0.0.0-20190626221950-04f50cda93cb

### CODE REVIEW

File: internal/oci/runtime_vm.go [hunk 1]
```diff
@@ -221,8 +221,8 @@ func (r *runtimeVM) startRuntimeDaemon(c *Container) error {
 		return err
 	}
 
-	cl := ttrpc.NewClient(conn)
-	cl.OnClose(func() { conn.Close() })
+	options := ttrpc.WithOnClose(func() { conn.Close() })
+	cl := ttrpc.NewClient(conn, options)
```
Comment (saschagrunert) [inline internal/oci/runtime_vm.go]: /cc @sboeuf

**Comment by giuseppe** [comment]
LGTM

**Pull request merged**